src/components/GatsbyLink.js

```diff
@@ -1,9 +1,9 @@
-/* eslint-disable no-unused-vars */
-import React from 'react'
+import * as React from 'react'
 import { Link } from 'gatsby'
-import linkResolver from '../utils/linkResolver'
 
-const GatsbyLink = (type, element, content, children, index) => {
+import { linkResolver } from '../utils/linkResolver'
```

Comment:
same here alias for utils

Reply:
Aliases are a form of indirection when the imports are short and understandable like this example (`../utils/linkResolver`). Keeping it relative allows users to understand where files are relative to the current file. Using aliases removes that wayfinding.

Since the other Gatsby example projects are not using aliases (for that reason), I think we should keep it consistent with relative imports.